src/ophyd_async/tango/core/_tango_transport.py

```diff
@@ -40,6 +28,18 @@
 )
 from tango.utils import is_array, is_binary, is_bool, is_float, is_int, is_str
 
+from ophyd_async.core import (
+    AsyncStatus,
+    Callback,
+    NotConnected,
+    SignalBackend,
```

Comment:
```suggestion
```

nit: Code scanning says this isn't required but I don't 100% believe if it the linting didn't catch it

Reply:
the AI is wrong...